tools/node-lint-md-cli-rollup/rollup.config.js

```diff
@@ -0,0 +1,48 @@
+import resolve from '@rollup/plugin-node-resolve';
+import commonjs from '@rollup/plugin-commonjs';
```

Comment:
these should be require instead of import.

Reply:
Taken care of, thanks.